Scenario/scenario_shacl.ttl

```diff
@@ -0,0 +1,439 @@
+
+scenario:EnvironmentalShape a sh:NodeShape ;
+    sh:property [ skos:example "32.33" ;
+            sh:datatype xsd:float ;
+            sh:description "Elevation of the sun"@en ;
+            sh:message "Validation of sunElevation failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "sunElevation"@en ;
+            sh:order 1 ;
+            sh:path scenario:sunElevation ],
+        [ skos:example "dry, moist, wetWithPuddles, lowFlooded, highFlooded" ;
+            sh:datatype xsd:string ;
+            sh:description "If street is wet or not"@en ;
+            sh:message "Validation of wetness failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "wetness"@en ;
+            sh:order 3 ;
+            sh:path scenario:wetness ],
+        [ skos:example "194.01" ;
+            sh:datatype xsd:float ;
+            sh:description "Azimuth of the sun"@en ;
+            sh:message "Validation of sunAzimuth failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "sunAzimuth"@en ;
+            sh:order 2 ;
+            sh:path scenario:sunAzimuth ],
+        [ skos:example "30m fog" ;
+            sh:datatype xsd:string ;
+            sh:description "List of environmental conditions"@en ;
+            sh:message "Validation of environmentConditions failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "environmentConditions"@en ;
+            sh:order 0 ;
+            sh:path scenario:environmentConditions ] ;
+    sh:targetClass scenario:Environmental .
+
+scenario:FormatShape a sh:NodeShape ;
+    sh:property [ skos:example "ASAM OpenSCENARIO" ;
+            sh:datatype xsd:string ;
+            sh:description "Format type definition"@en ;
+            sh:in ("ASAM OpenSCENARIO") ;
+            sh:maxCount 1 ;
+            sh:message "Validation of type failed!"@en ;
+            sh:name "type"@en ;
+            sh:order 0 ;
+            sh:path scenario:type ],
+        [ skos:example "1.5" ;
+            sh:datatype xsd:string ;
+            sh:description "Version of data format"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of version failed!"@en ;
+            sh:name "version"@en ;
+            sh:order 1 ;
+            sh:path scenario:version ] ;
+    sh:targetClass scenario:Format .
+
+scenario:GeneralShape a sh:NodeShape ;
+    sh:property [ skos:example "Functional, Logical, Concrete" ;
+            sh:datatype xsd:string ;
+            sh:description "Pegasus type of scenario"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of type failed!"@en ;
+            sh:name "type"@en ;
+            sh:order 0 ;
+            sh:path scenario:type ],
+        [ skos:example "cut-in, cut-out" ;
+            sh:datatype xsd:string ;
+            sh:description "Maneuver type in the sense of a functional scenario"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of maneuver failed!"@en ;
+            sh:name "maneuver"@en ;
```

Comment:
can you add an `sh:in` here?

Reply:
The maneuver-properties should be handle with OpenLABEL. Therefore, I would suggest to leave it out here.

Reply:
ok, resolved